scripts/analysis.py

```diff
@@ -1091,32 +1034,229 @@ def plot_power(cursor):
+        # get power / number timeseries
+        plot_power(cur)
 
+        # get reprocessing waste
+        reprocess_waste = commodity_in_out_facility(
+            cur, 'separations', ['mox_reprocess_waste', 'reprocess_waste'], True, False, False)
+        stacked_bar_chart(reprocess_waste, timestep, 'Years',
+                          'Mass [MTHM]', 'Reprocess Waste vs Time', 'reprocess_waste', init_year)
+
+"""
+        dictionary = collections.OrderedDict()
+        dictionary['natural_uranium'] = nat_u_timeseries(cur)
+        stacked_bar_chart(dictionary, timestep,
+                          'Years', 'U Utilization Factor',
+                          'U Utilization vs Time',
+                          'u_util', init_year)
+        combined = commodity_in_out_facility(cur, 'separations', ['mox_Pu'], True, False, True)
+        stacked_bar_chart(combined, timestep, 'Years', 'Mass[MTHM]', 'Total reprocessed Pu vs Time', 'combined', init_year)
+        mox_Pu = commodity_in_out_facility(cur, 'separations', ['mox_Pu'], True, False, True)
+        stacked_bar_chart(mox_Pu, timestep, 'Years', 'Mass[MTHM]', 'reprocessed Pu from spent MOX vs Time', 'reprocessed', init_year)
+        uox_Pu = commodity_in_out_facility(cur, 'separations', ['uox_Pu'], True, False, True)
+        stacked_bar_chart(uox_Pu, timestep, 'Years', 'Mass[MTHM]', 'reprocessed Pu from spent UOX vs Time', 'uox_reprocessed', init_year)
+        #stacked_bar_chart(get_trade_dict(cur, 'uox_reprocessing', 'mox_fuel_fab', True, True), timestep,
+        #                  'Years', 'Mass [MTHM]', 'reprocessed Pu outflux vs Time', 'ahhhhh', init_year)
+
+# Europe History Case Only
+        #tailings = commodity_in_out_facility(cur, 'uox_mixer', ['tailings'], True)
+        # stacked_bar_chart(tailings, timestep,
+        #                  'Year', 'Mass [MTHM]',
+        #                  'Tailings vs Time',
+        #                  'tailings',
+        #                  init_year)
+        #uox_pu = commodity_from_facility(cur, 'separations', ['uox_Pu'])
+        # stacked_bar_chart(uox_pu, timestep,
+        #                  'Year', 'Mass [MTHM]',
+        #                  'Pu output (UOX) vs Time',
+        #                  'tailings',
+        #                  init_year)
+
+        #mox_pu = commodity_from_facility(cur, 'separations', ['mox_Pu'])
+        # stacked_bar_chart(mox_pu, timestep,
+        #                  'Year', 'Mass [MTHM]',
+        #                  'Pu output (MOX) vs Time',
+        #                  'tailings',
+        #                  init_year)
+        fuel_dict = where_comm(
+            cur, 'mox', ['mox_uox_fuel_fab', 'mox_mox_fuel_fab'])
         stacked_bar_chart(fuel_dict, timestep,
                           'Years', 'Mass[MTHM]',
                           'Total Fuel Mass vs Time',
-                          'total_fuel',
+                          'where_fuel',
```

Comment:
We've removed all these lines and decided to store them in a separate file depending on the simulation.